### DISCUSSION TITLE
[Discussion]: Project Becoming Dead!

### DISCUSSION THREAD

**Discussion by IzumiSenaSora** [aivanovski/keepassvault#283] [General]
### Overview

Hi!

Thanks for this Awesome smooth experience KeePass Client app!

I love this app look,design,smoothness,data history everything!

But the thing i fear is KeePass client projects get unmaintained or dead soon!

That's why i am asking you about this project future!

**Comment by aivanovski** [comment]
Hi,

Initially I've created this project for myself, as I need an application that can read KeePass database and keep it in sync with some server. Mostly because KeePassDX didn't allow to do that and other apps were outdated. So, the main driving force of this project is my personal need to use KeePass database.

There is a [board](https://github.com/users/aivanovski/projects/2/views/1) with all features that I'm going to implement in the future. But as I'm the only one active maintainer it is moving forward slowly.

**Comment by IzumiSenaSora** [comment]
> Hi,
> 
> Initially I've created this project for myself, as I need an application that can read KeePass database and keep it in sync with some server. Mostly because KeePassDX didn't allow to do that and other apps were outdated. So, the main driving force of this project is my personal need to use KeePass database.
> 
> There is a [board](https://github.com/users/aivanovski/projects/2/views/1) with all features that I'm going to implement in the future. But as I'm the only one active maintainer it is moving forward slowly.

I hope this project survive and get Lot's of maintainers!
And i pray for you so that you always stay healthy and strong and keep up this project!
Thanks a lot!

**Comment by RonnyG2121** [comment]
Hello.

I use this application and I like it.

But I'd like to know if it's possible to add translations to the project.

I speak spanish.

Thanks.
